src/components/forms/FileInput/FileInput.stories.tsx

```diff
@@ -108,13 +108,15 @@ export const disabled = (): React.ReactElement => (
   </FormGroup>
 )
 
-export const withCustomHandlers = (argTypes): React.ReactElement => {
+export const withRefAndCustomHandlers = (argTypes): React.ReactElement => {
   const [files, setFiles] = useState<FileList>()
-  const fileInputRef = useRef<HTMLInputElement>()
+  const fileInputRef = useRef<FileInputRef>()
```

Comment:
FYI Typescript in our project (may be a stricter setting) doesn't like this.   Had to set it up as `const fileInputRef = useRef<FileInputRef>(null)`

`Type 'FileInputRef | undefined' is not assignable to type 'FileInputRef | null'.
        Type 'undefined' is not assignable to type 'FileInputRef | null'.ts(2322)`

That conflict is with the type for `FileRefInput['input']'`

Reply:
I honestly can't remember why I used `null` instead of `undefined` (on [this line](https://github.com/trussworks/react-uswds/pull/1165/files#diff-39e22af5b23c82a7bd3b2c4dde1d5835d1ab01ab4f1173b9290c9a42b1477e0dR36)). I'm changing to `undefined`, let me know if that resolves!